packages/docusaurus/src/client/serverEntry.tsx

```diff
@@ -83,19 +83,23 @@ async function doRender(locals: Locals & {path: string}) {
   const modules = new Set<string>();
   const context = {};
 
+  const helmetContext = {};
+
   const linksCollector = createStatefulLinksCollector();
   const appHtml = ReactDOMServer.renderToString(
     <Loadable.Capture report={(moduleName) => modules.add(moduleName)}>
-      <StaticRouter location={location} context={context}>
-        <ProvideLinksCollector linksCollector={linksCollector}>
-          <App />
-        </ProvideLinksCollector>
-      </StaticRouter>
+      <HelmetProvider context={helmetContext}>
```

Comment:
probably worth applying strict mode here too for security, even if I doubt it will be very useful (only enabled in prod) we may enable some SSR in development mode in the future.

See also https://twitter.com/sebastienlorber/status/1481295361016745984